.azure/pipelines/Phonebook.preview.yml

```diff
@@ -210,12 +210,12 @@ jobs:
     displayName: 'Delay Deletion of Preview Environment'
     pool: server
     dependsOn: deploy_preview
-    timeoutInMinutes: 725
-    cancelTimeoutInMinutes: 725
+    timeoutInMinutes: 1450
```

Comment:
why 24h and 16 min? Why not 24h?

Reply:
Because Azure is very precise... If you delay job is 24h and your timeout is 24h the job will fail. So you have to add a little safety-margin.

Reply:
added a comment for clarity